plugins/topology/src/components/Topology/TopologyViewWorkloadComponent.tsx

```diff
@@ -49,6 +54,15 @@ const TopologyViewWorkloadComponent = ({
     removeSelectedIdParam,
   ] = useSideBar();
 
+  const topologyViewPermissionResult = usePermission({
+    permission: topologyViewPermission,
+  });
+
+  const catalogEntityPermissionResult = usePermission({
+    permission: catalogEntityReadPermission,
+    resourceRef: catalogEntityReadPermission.resourceType,
```

Comment:
@PatAKnight I removed lgtm so that @AndrienkoAleksandr can also reveiw the PR but it still got merged. :P
Last time I checked with @AndrienkoAleksandr this is not the way catalogEntityReadPermission authorization works as it expects the complete resourceRef like `user:default/div`. I have opened one UI issue regarding this as when conditions are added to a role of admin this /api/permission/authorize returns this error for catalogEntityReadPermission : `
"Entity reference \"catalog-entity\" had missing or empty kind (e.g. did not start with \"component:\" or similar)"`